### PR TITLE
Automated test scripts for chrome.

### PR DESCRIPTION
**Pull request by polarbehrens**
automatedTestChrome2 is for normal users and automatedTestChrome is for superusers. Uses helium api which really sucks. In the process of switching to a different api.

### CODE REVIEW

**Review by JaredTCan — APPROVED** [review]
Looks good!

**Comment by AparaV** [comment]
Let's wait until we have completed this before merging @JaredTCan @polarbehrens

File: automatedTestChrome.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+def logout():
+    # used to log out normal user to test superuser
+    click("Logout")
+
+def superUserTest():
+
+    # logs in superuser
+    go_to("http://73.229.199.171/")
+    write("god", into="Username")
```
Comment (AparaV) [inline automatedTestChrome.py:13]: We should store these credentials as environment variables or as a `.config` file that we don't track

File: automatedTestChrome2.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+from helium.api import *
+import time
+
+# test login info
```
Comment (AparaV) [inline automatedTestChrome2.py:4]: Same. We should store these as environment variables or as a `.config` file

File: automatedTestChrome.py (hunk 1 above)
Comment (polarbehrens) [reply automatedTestChrome.py:13]: ok ill change them to environment variables

**Comment by polarbehrens** [comment]
Switched to a different api on pr #18. Closing

**Pull request closed without merging**